src/store/participantsStore.js

```diff
@@ -56,6 +56,20 @@ function emitUserStatusUpdated(participant) {
 	}
 }
 
+/**
+ * Get connection failed error message as per error code.
+ *
+ * @param {*} code Error message code
+ *
+ * @return {string} Error message
+ */
+function getConnectionFailedErrorMessage(code) {
+	if (code === 'consent') {
+		return t('spreed', 'Recording consent is required')
+	}
+	return t('spreed', 'Oops, something went wrong!')
+}
```

Comment:
I'm not sure this belongs to the store. Maybe we'll move it to EmptyCallView component (in computed `message`, where it initially was)?

Reply:
@Antreesy Did you mean to write this logic directly in the `message` function of `EmptyCallView` component, like this? - https://github.com/nextcloud/spreed/pull/13133/files#diff-3d0a2e872c65aa3113ac800fa38a10dbe0d15f932a9ebbd9c14adb34f3018dc7R158-R164

Reply:
Yeah, that's more than enough =)